migrations/educandu-2024-04-04-02-add-documentRatings-collection.js

```diff
@@ -0,0 +1,20 @@
+export default class Educandu_2024_04_04_02_add_documentRatings_collection {
```

Comment:
This is a reminder to increase the counter after https://github.com/educandu/educandu/pull/1136 got merged